src/OutputReaders/adapted_field_time_series.jl

```diff
@@ -0,0 +1,17 @@
+using Adapt
+
+struct AdaptedFieldTimeSeries{T, D, χ} <: AbstractArray{T, 4}
```

Comment:
```suggestion
struct GPUAdaptedFieldTimeSeries{T, D, χ} <: AbstractArray{T, 4}
```

It's important to put that to understand the purpose of this